docs/source/en/model_doc/aya_vision.md

```diff
@@ -102,132 +50,61 @@ messages = [
 
-For processing batched inputs (multiple conversations at once):
+</hfoption>
 
-```python
-from transformers import AutoProcessor, AutoModelForImageTextToText
-import torch
+</hfoptions>
 
-model_id = "CohereForAI/aya-vision-8b"
+Quantization reduces the memory footprint of large models by representing weights at lower precision. Refer to the [Quantization](https://huggingface.co/docs/transformers/main/en/main_classes/quantization) overview for supported backends.
 
-processor = AutoProcessor.from_pretrained(model_id)
-model = AutoModelForImageTextToText.from_pretrained(
-    model_id, device_map="cuda:0", torch_dtype=torch.float16
-)
+## Notes
 
-# Prepare two different conversations
-batch_messages = [
-    # First conversation with a single image
-    [
-        {
-            "role": "user",
-            "content": [
-                {"type": "image", "url": "https://llava-vl.github.io/static/images/view.jpg"},
-                {"type": "text", "text": "Write a haiku for this image"},
-            ],
-        },
-    ],
-    # Second conversation with multiple images
-    [
-        {
-            "role": "user",
-            "content": [
-                {
-                    "type": "image",
-                    "url": "https://cdn.britannica.com/61/93061-050-99147DCE/Statue-of-Liberty-Island-New-York-Bay.jpg",
-                },
-                {
-                    "type": "image",
-                    "url": "https://thumbs.dreamstime.com/b/golden-gate-bridge-san-francisco-purple-flowers-california-echium-candicans-36805947.jpg",
-                },
-                {
-                    "type": "text",
-                    "text": "These images depict two different landmarks. Can you identify them?",
-                },
-            ],
-        },
-    ],
-]
-
-# Process each conversation separately and combine into a batch
-batch_inputs = processor.apply_chat_template(
-    batch_messages, 
-    padding=True, 
-    add_generation_prompt=True, 
-    tokenize=True, 
-    return_dict=True, 
-    return_tensors="pt"
-).to(model.device)
+- Use explicit language tokens (e.g. `<en>`, `<fr>`, `<hi>`) in prompts to reliably set the output language—otherwise it infers from input, which may be less precise.
 
-# Generate responses for the batch
-batch_outputs = model.generate(
-    **batch_inputs,
-    max_new_tokens=300,
-    do_sample=True,
-    temperature=0.3,
-)
+- You can pass a list of messages to the `pipeline` or manually format them via `apply_chat_template`, allowing multiple images or prompts in a single forward pass.
 
-# Decode the generated responses
-for i, output in enumerate(batch_outputs):
-    response = processor.tokenizer.decode(
-        output[batch_inputs.input_ids.shape[1]:], 
-        skip_special_tokens=True
-    )
-    print(f"Response {i+1}:\n{response}\n")
-```
+- The model uses `use_cache=True` by default, so repeated `generate(...)` calls retain past key/value states and speed up decoding for long outputs.
```

Comment:
Add code snippet from [here](https://huggingface.co/docs/transformers/main/en/model_doc/aya_vision#multiple-images-and-batched-inputs)

```suggestion
- The example below demonstrates inference with multiple images.

   ```py
   ```
```